src/crimson/os/seastore/omap_manager.h

```diff
@@ -114,37 +114,43 @@ class OMapManager {
     size_t max_result_size = 128;
 
     /// true denotes behavior like lower_bound, upper_bound otherwise
-    bool inclusive = false;
+    /// range start behavior
+    bool first_inclusive = false;
+    /// range end behavior
+    bool last_inclusive = false;
 
     omap_list_config_t(
       size_t max_result_size,
-      bool inclusive)
+      bool first_inclusive,
+      bool last_inclusive = false)
```

Comment:
no need to set default here as the `last_inclusive` member already has the default value.